app/helpers/application_helper.rb

```diff
@@ -409,6 +404,12 @@ def edit_inline(object, property, options = {})
     end
   end
 
+  def documentation_url(section = "")
+    base_url = "http://www.theforeman.org/documentation.html"
+    base_url = "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#" + section unless section.empty?
```

Comment:
There's no point in setting base_url on line 408 when `section.present?`, so just put it into an if/else block, like the one in the original documentation_button.